tcs-service/src/test/java/com/transformuk/hee/tis/tcs/service/listener/person/TrainingNumberElasticSearchEventListenerTest.java

```diff
@@ -0,0 +1,108 @@
+package com.transformuk.hee.tis.tcs.service.listener.person;
```

Comment:
May be the same here

Reply:
I can't see any license declaration in any of other tcs files, so think it's not needed.